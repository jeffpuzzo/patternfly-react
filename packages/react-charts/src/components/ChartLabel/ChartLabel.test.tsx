import * as React from 'react';
import { render } from '@testing-library/react';
import { ChartLabel } from './ChartLabel';

Object.values([true, false]).forEach(() => {
  test('ChartLabel', () => {
    const view = render(<ChartLabel />);
    expect(view.container.outerHTML).toMatchSnapshot();
  });
});

test('renders component text', () => {
  const view = render(<ChartLabel text="This is a test" />);
  expect(view.container.outerHTML).toMatchSnapshot();
});
