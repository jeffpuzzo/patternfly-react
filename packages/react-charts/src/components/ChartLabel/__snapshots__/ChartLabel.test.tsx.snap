// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChartLabel 1`] = `"<div></div>"`;

exports[`ChartLabel 2`] = `"<div></div>"`;

exports[`renders component text 1`] = `"<div><text direction=\\"inherit\\" dx=\\"0\\" x=\\"0\\" y=\\"4.97\\"><tspan x=\\"0\\" dx=\\"0\\" dy=\\"0\\" text-anchor=\\"start\\" style=\\"font-family: var(--pf-chart-global--FontFamily); font-size: 14px; letter-spacing: var(--pf-chart-global--letter-spacing); fill: #252525; stroke: transparent;\\">This is a test</tspan></text></div>"`;
